settingshelper.py

```diff
@@ -254,6 +254,7 @@ def configure_sentry(base_dir, server_env, pub_key, priv_key, project_id):
         'processors': (
             'raven.processors.SanitizePasswordsProcessor',
             'raven.processors.RemovePostDataProcessor',
+            'corehq.util.sentry.HQSanitzeSystemPasswordsProcessor',
```

Comment:
Do we need this one and `SanitizePasswordsProcessor`? Since you're overriding `SanitizePasswordsProcessor` and calling `super` I think we only need one.

Reply:
you're totally right, thanks

Reply:
https://github.com/dimagi/commcare-hq/pull/19159/commits/011af398583422b3abe5bf2924fc7143a7b0754c